give-a-talk.md

```diff
@@ -1,40 +1,46 @@
 ---
 layout: page
-title: Give a Lightning Talk
+title: We want you to give a talk!
 ---
 {% include JB/setup %}
 
-### We want you to give a talk!
+If you have something JavaScript related to share with your peers that you could say/show in less than 5 minutes, **we want to hear you give a talk**!
```

Comment:
Do we want to call out that if you have a longer talk as well we might still have an opening?  Or does that complicate things needlessly?